doc/datasets/rcv1.rst

```diff
@@ -0,0 +1,54 @@
+
+.. _rcv1:
+
+RCV1 dataset
+============
+
+Reuters Corpus Volume I (RCV1) is an archive of over 800,000 manually categorized newswire stories made available by Reuters, Ltd. for research purposes. The dataset is extensively described in [1]_.
+
+:func:`sklearn.datasets.fetch_rcv1` will load the vector version of RCV1-v2 dataset::
+
+    >>> from sklearn.datasets import fetch_rcv1
```

Comment:
I think you should protect the docstring to avoid downloading the data on travis no?